### PR TITLE
Change URL: add_movie: `_` to `-` and add_review: `_` to `-` 

### PR DESCRIPTION
**Pull request by ADITIRAI290698**

### CODE REVIEW

**Comment by ADITIRAI290698** [comment]
@AbrarNitk please review the PR and merge it.

File: dj/ui/add-movie.ftd [hunk 1]
```diff
@@ -323,7 +323,7 @@ $on-click$: message-host $add-movie-api
 -- object add-movie-api:
 function: http
 method: post
-url: /add-movie/
+url: /add_movie/
```
Comment (AbrarNitk) [inline dj/ui/add-movie.ftd]: It should be `add-movie`

**Review by AbrarNitk — CHANGES_REQUESTED** [review]

**Comment by ADITIRAI290698** [comment]
@AbrarNitk  Done the required changes. Please check and please merge the PR.

**Comment by AbrarNitk** [comment]
This PR is no longer required.

**Pull request closed without merging**